### ISSUE TITLE
Bug in API - calling undefined function

### ISSUE THREAD

**Issue by pweisensee** [expo/xdl#4]
I'm getting an error when calling `exp detach` with `exp@37.0.0` that seems to be related to a fix 5 days ago: [https://github.com/exponent/xdl/commit/d975ee1a4a507b486bf470411cb95a5705d4693f](https://github.com/exponent/xdl/commit/d975ee1a4a507b486bf470411cb95a5705d4693f)

I can provide more info if needed, see errors from console below:

```
Your project looks good!
Validating project manifest...
Downloading iOS code...

/xdl/src/Api.js:155
          progressFunction(100);
          ^
TypeError: progressFunction is not a function
    at WriteStream.<anonymous> (/xdl/src/Api.js:155:11)
    at emitNone (events.js:91:20)
    at WriteStream.emit (events.js:185:7)
    at finishMaybe (_stream_writable.js:515:14)
    at afterWrite (_stream_writable.js:389:3)
    at onwrite (_stream_writable.js:379:7)
    at WritableState.onwrite (_stream_writable.js:90:5)
    at fs.js:2137:5
    at FSReqWrap.wrapper [as oncomplete] (fs.js:747:5)
```

**Comment by jesseruder** [comment]
Thanks will look at this now.

**Comment by jesseruder** [comment]
Should be fixed in `exp@37.0.1`

**Comment by pweisensee** [comment]
Awesome, thanks for the quick response!
